bundle/src/main/java/com/adobe/acs/commons/wcm/impl/SiteMapServlet.java

```diff
@@ -233,6 +239,22 @@ protected void doGet(SlingHttpServletRequest request, SlingHttpServletResponse r
         return allAssetFolders;
     }
 
+    private String applyUrlRewrites(String url) {
+        for (String rewrite : urlRewrites) {
+            if (!rewrite.contains(":") || rewrite.startsWith(":")) {
+                continue;
+            }
+            String[] tokens = rewrite.split(":");
+            String path = tokens[0];
+            String replacement = tokens.length > 1 ? tokens[1] : "";
+            if (url.contains(path)) {
```

Comment:
Should this be "contains" or "startsWith" ?  .. It might be an outlier case, but couldn't `contains` result in false positives?

Reply:
Addressed in the latest commit